src/pages/MyPage/MyPage.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import './MyPage.scss';
+
+export default class MyPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userRatedData: [],
+    };
+  }
+
+  componentDidMount() {
+    let token = localStorage.getItem('TOKEN') || '';
+    fetch(
+      '/data/userArchiveData.json'
+      //  {
+      //   headers: {
+      //     Authorization: token,
+      //   },
+      // }
+    )
+      .then(res => {
+        if (res.status === 200) {
+          return res.json();
+        }
+      })
+      .then(userRatedData => {
+        this.setState({
+          userRatedData: userRatedData['result'],
```

Comment:
```suggestion
          userRatedData: userRatedData.result
```
동적으로 접근할 거 아니고, 특수문자도 없으니까 dot notation으로 충분하지 않을까요?

Reply:
아!!! 수정 완료했습니다 👍🏻